core/build.sbt

```diff
@@ -485,6 +502,11 @@ lazy val commonSettings = Seq(
   scalacOptions in (Test, console) := (scalacOptions in (Compile, console)).value,
 )
 
+val altBintrayRepository = settingKey[Option[String]]("Overrides the default bintray repository if defined")
+ThisBuild / altBintrayRepository := None
```

Comment:
Do we still use the `altBintrayRepository` setting ? It was used to push nightly builds to a different bintray repo, but we don't do this anymore.
I don't see a reference to it in the release scripts.

Reply:
yeah @skyluc  I think we can remove it and set the repo name straight to `cloudflow`

Reply:
removed `altBintrayRepository` ..